nat-lab/tests/uniffi/serialization.py

```diff
@@ -0,0 +1,62 @@
+import inspect
+import Pyro5.api  # type: ignore
+
+
+def init_serialization(libtelio):
+    all_members = inspect.getmembers(libtelio, inspect.isclass)
+    public_members = []
+    for name, cls in all_members:
```

Comment:
Maybe this is okay, I don't know for sure. But we are going only 1 level deep here. What if we switched to calling this recursively ? IMO this feels a bit rigid.

Reply:
When I switch to calling it recursively the tests don't start because it enters an infinite loop. Basically, everything is an object, `inspect.isclass` returns more than I think it should, so there is no end to getting nested types